sdk/python/kfp/components/_python_op.py

```diff
@@ -723,8 +723,21 @@ def add_multiply_two_numbers(a: float, b: float) -> NamedTuple('DummyName', [('s
     Path(output_component_file).write_text(component_yaml)
 
 
-def func_to_container_op(func, output_component_file=None, base_image: str = None, extra_code='', packages_to_install: List[str] = None, modules_to_capture: List[str] = None, use_code_pickling=False):
-    '''Converts a Python function to a component and returns a task (:class:`kfp.dsl.ContainerOp`) factory.
+def func_to_container_op(
+    func,
+    output_component_file = None,
+    base_image: str = None,
+    extra_code = '',
+    packages_to_install: List[str] = None,
+    modules_to_capture: List[str] = None,
+    use_code_pickling = False):
```

Comment:
Some args are typed but not others, what is the standard? Should all be typed?

Reply:
Yes we should. Will add it. Thanks for pointing!

Reply:
Done.